tests/test_private_registry.py

```diff
@@ -0,0 +1,59 @@
+from tests.helm_template_generator import render_chart
+import jmespath
+
+
+def test_private_repository_image_names_the_same_as_public_ones():
+    """test that image names dont change when using a custom repo because that breaks
+    pull-through caching proxies in use by various customers"""
+    repository = "quay.io/astronomer"
+    public_repo_docs = render_chart()
+    private_repo_docs = render_chart(
+        values={
+            "global": {"privateRegistry": {"enabled": True, "repository": repository}}
+        },
+    )
+    # should be same number of images regardless of where they come from
+    assert len(public_repo_docs) == len(private_repo_docs)
+    search_string = "spec.template.spec.containers[*].image"
+    differtly_named_images = []
```

Comment:
Got a typo in several places in both tests
```suggestion
    differently_named_images = []
```